src/main/java/com/codingbat/exercise1/Problem2_Diff21.java

```diff
@@ -0,0 +1,18 @@
+package com.codingbat.exercise1;
+
+public class Problem2_Diff21 {
+
+    public int diff21(int n) {
+
+        int difference = 21 - n;

```

Comment:
Can we move this 21 value to a static variable so that we can change it if required?